Sources/ComposableArchitecture/TestSupport/XCTestShims.swift

```diff
@@ -0,0 +1,3 @@
+typealias XCTCurrentTestCase = @convention(c) () -> AnyObject
+typealias XCTFailureHandler
+  = @convention(c) (AnyObject, Bool, UnsafePointer<CChar>, UInt, String, String?) -> Void
```

Comment:
Moved these into their own file since swift-format breaks on them and we still would like TestStore.swift to be formatted.

Reply:
Actually turns out that swift-format tripped on them only when they were referenced inline `(@convention(c) (...) -> ...).self`.

They format fine as type aliases!